pkg/agent/apiserver/handlers/bgproute/handler.go

```diff
@@ -72,11 +73,13 @@ func HandleFunc(bq querier.AgentBGPPolicyInfoQuerier) http.HandlerFunc {
 
 		var bgpRoutesResp []apis.BGPRouteResponse
 		for bgpRoute := range bgpRoutes {
```

Comment:
now I feel like having the `ipv4Routes, ipv6Routes bool` parameters in the `GetBGPRoutes` querier method is not necessary or desirable, and that we should do all the filtering here, like you do for the route type. What do you think? It would be more consistent and simplify the logic in `GetBGPRoutes`.

Reply:
I felt the same while making these changes. I think we can also do the same for `GetBGPPeerStatus` as well. I will update the changes for both in this PR.